localization/english/modifiers/wc_holy_light_orders_modifiers_l_english.yml

```
@@ -0,0 +1,12 @@
+﻿l_english:
+ ### Enforce the Brotherhood of Northshire ###
+ spy_the_brotherhood_modifier:0 "Spy the Brotherhood"
+ spy_the_brotherhood_modifier_desc:0 "This character as infiltrated the Brotherhood of Northshire to gather intelligence and anticipate their moves."
```

```suggestion
 spy_the_brotherhood_modifier_desc:0 "This character as infiltrated the Brotherhood of Northshire to gather intelligence and anticipate their moves."
```

this is redundant, what else does spying allow you to do? if nothing, just delete ` and anticipate their moves`